pkgs/development/python-modules/pyqodeng-angr/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  future,
+  pygments,
+  pyside6,
+  pytestCheckHook,
+  pytest-xdist,
+  qtpy,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "pyqodeng-angr";
+  version = "0.0.13";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "angr";
+    repo = "pyqodeng";
+    tag = "v${version}";
+    hash = "sha256-t4LcPVQfktAaTqTr9L2VDCEHbSO7qxCvUDz6rj0Zre4=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-quiet 'PySide6-Essentials' 'PySide6'
+  '';
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pygments
+    future
```

Comment:
`future` no longer works with `python3.13`, so this needs angr/pyqodeng#15 (either in patch form or by waiting for a release).

Reply:
I just noticed that `pyqodeng-angr` got broken out into a new PR and merged, so this should be fixed in the existing package instead.